cmd/server/cadence/cadence.go

```diff
@@ -145,8 +148,13 @@ func getRootDir(c *cli.Context) string {
 	return dirpath
 }
 
-func constructPath(dir string, file string) string {
-	return dir + "/" + file
+// constructPathIfNeed would append the dir as the root dir
+// when the file wasn't absolute path.
+func constructPathIfNeed(dir string, file string) string {
+	if !filepath.IsAbs(file) {
+		return dir + "/" + file
+	}
+	return file
 }
 
 // BuildCLI is the main entry point for the cadence server
```

Comment:
Can you also fix line 178? 
maybe :
`config dir path relative to root, or an absolute path`.

Reply:
Fixed it, sorry for missing that.